backend/routes/courses.js

```diff
@@ -20,6 +20,48 @@ router.get("/getCourseByTag", async function(req, res, next) {
   }
 });
 
+/*
+* @api {post} /createCourse
+* @apiDescription This endpoint will add a course and an associated user
+* @apiParam (body) {string} courseCode, {string} title, {string} departement
+* @apiExample {curl} Example usage:
+* Http: 
+	POST /courses/createCourse HTTP/1.1
+	Host: localhost:3001
+	Content-Type: application/json
+	{
+		"courseCode": "ECSE 428",
+		"title": "Software Engineering Practice",
+		"departement" : "ECSE"
+	}
+* Curl:
+	curl -X POST \
+	http://localhost:3001/courses/createCourse\
+	-H 'Content-Type: application/json' \
+	-d '{
+		"courseCode": "ECSE 428",
+		"title": "Software Engineering Practice",
+		"departement" : "ECSE"
+	}'
+*
+* @returns true if course was added successfully or false otherwise
+*
+* @author: Mathieu Savoie
+*/
+router.post('/createCourse', function(req, res, next) {
+  const courseCode = req.body.courseCode;
+  const title = req.body.title;
+  const departement = req.body.departement;
+  const phasedOut = req.body.phasedOut
```

Comment:
you have phased out here, but not in the params specified in the comment above. Careful about this. Works out well in this case though since we should remove it from params anyways (or put it as a optional param by allowing it to be indefined, up to you )